ci/ray_ci/tester_container.py

```diff
@@ -33,6 +37,7 @@ def __init__(
         used to run tests in a distributed fashion.
         :param shard_ids: The list of shard ids to run. If none, run no shards.
         """
+        self.team = team
```

Comment:
the team info can be parsed from the tag fields of the test log, right? so you do not need to send it inside here?

this is now restricting a test container to one and only one team, which feels unnecessary.

Reply:
yes it can be obtained from the log but the info should come from only one source of truth; i intentionally make the `oncall` as the required param in `test_in_docker` so tests are even runnable without a proper owner.

Reply:
not really, parsing the team tag syntax is the job of `test_in_docker`; if the syntax is changed to ray_team:core, it's the work between `test_in_docker` and bazel rule to deal with it.

Reply:
talked offline, agreed to move this into `run_tests`, so that is is not on the class instance construction path, which is harder to change/expand when required.